<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogSelectRaces</class>
 <widget class="QDialog" name="DialogSelectRaces">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>441</width>
    <height>141</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Select Races</string>
  </property>
  <widget class="QPushButton" name="pushButton_ReplaceRace">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>90</y>
     <width>141</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>Replace</string>
   </property>
   <property name="icon">
    <iconset resource="../Resources/InteractiveGameTweaker.qrc">
     <normaloff>:/heroes/bahgtru.gif</normaloff>:/heroes/bahgtru.gif</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>32</width>
     <height>32</height>
    </size>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_ReplaceFromRace">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>10</y>
     <width>201</width>
     <height>30</height>
    </rect>
   </property>
   <property name="maxVisibleItems">
    <number>30</number>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_Plane">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>10</y>
     <width>101</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_ReplaceByRace">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>50</y>
     <width>201</width>
     <height>30</height>
    </rect>
   </property>
   <property name="maxVisibleItems">
    <number>30</number>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>46</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Replace</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>10</y>
     <width>31</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>on</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>50</y>
     <width>46</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>by</string>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="../Resources/InteractiveGameTweaker.qrc"/>
 </resources>
 <connections/>
</ui>
